spec/services/nytimes_service_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+RSpec.describe NytimesService do
+
+	describe "articles" do
+		it "saves articles in db" do
+			VCR.use_cassette("articles") do
+        NytimesService.new.articles
+			end
+
+			expect(Article.first.title).to(
+				eq("California Imposes Water Restrictions, in First for State")
+			)
+			expect(Article.first.url).to(
+				eq("http://www.nytimes.com/2015/04/02/us/california-imposes-first-" +
+					 "ever-water-restrictions-to-deal-with-drought.html")
+			)
+			expect(Article.first.abstract).to(
+				eq("The move by Gov. Jerry Brown comes as California’s drought has " +
```

Comment:
Indent the first parameter one step more than the previous line.<br>Tab detected.